files/jquery.mixitup/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "mixitup",
+  "repo": "patrickkunka/mixitup",
+  "files": {
+    "include": ["jquery.mixitup.js", "jquery.mixitup.min.js"]
```

Comment:
There are no such files here https://github.com/patrickkunka/mixitup/tree/v2.1.11
The bot will fail